src/Functional.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+
+const todoList = [
+    'Do React course',
+    'Go Shopping',
+    'Read',
+];
+
+function Functional() {
+    const [todo, setTodo] = useState(false);
+
+    const handleChange = (event) => {
+        setTodo(event.target.value);
+    } 
+
+    return (
+        <div style={{margin: '5px', padding: '5px'}}>
+            <h2>Todo List</h2>
+            {todoList.map((item, key) => {
+                return (
+                    <>
+                        <input
+                            type="radio"
+                            id={item}
+                            name="todo"
+                            value={key}
+                            onChange={handleChange}
+                        />
+
+                        <label for={item}>{item}</label><br />
```

Comment:
It's forbidden to use `for` as JSX attribute. Use `htmlFor` instead
[Docs](https://reactjs.org/docs/dom-elements.html#htmlfor)
It's like `class` in HTML. In JSX it's `className`